projects/adf5611/README.rst

```diff
@@ -0,0 +1,235 @@
+    +-----+-------------------+------------------------------------------+
+
+No-OS Build Setup
+-----------------
+
+Please see: https://wiki.analog.com/resources/no-os/build
+
+No-OS Supported Examples
+------------------------
+
+The initialization data used in the examples is taken out from:
+`Project Common Data Path <https://github.com/analogdevicesinc/no-OS/tree/main/projects/adf5611/src/common>`_
+
+The macros used in Common Data are defined in platform specific files found in:
+`Project Platform Configuration Path <https://github.com/analogdevicesinc/no-OS/tree/main/projects/adf5611/src/platform>`_
+
+Basic example
+^^^^^^^^^^^^^
+
+This example initializes the ADF5611 or ADF5612 with the configurations 
+provided in the  above mentioned common files and applies them to the IC. 
+By default reference frequency is provided by the local oscillator of the 
+board, the differential output path is powered down and the RF Output power 
+level is set to 3. Subsequently the example sets a test frequency of 12 GHz.
+
+
+In order to build the basic example make sure you are using this command:
+
+.. code-block:: bash
+
+    make BASIC_EXAMPLE=y
+
+IIO example
+^^^^^^^^^^^
+
+This project provide an IIOD demo for the ADF5611 and ADF5612. It launches an 
+IIOD server on the mother platform it runs on allowing the user to connect 
+via an IIOD client. Using IIO-Oscilloscope, the user can configure the ADF5611 
+or ADF5612. 
+
+If you are not familiar with ADI IIO Application, please take a look at:
+`IIO No-OS <https://wiki.analog.com/resources/tools-software/no-os-software/iio>`_
+
+The No-OS IIO Application together with the No-OS IIO ADF5611 driver take care
+of all the back-end logic needed to setup the IIO server.
+
+This example initializes the IIO device and calls the IIO app as shown in:
+`IIO Example <https://github.com/analogdevicesinc/no-OS/tree/main/projects/adf5611/src/examples/iio_example>`_
+
+In order to build the IIO project make sure you you are using this command:
+
+.. code-block:: bash
+
+    make IIO_EXAMPLE=y
+
+
+No-OS Supported Platforms
+-------------------------
+
+Mbed Platform
+^^^^^^^^^^^^^
+
+**Used hardware**
+
+* `EVAL-ADF5611 <https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/eval-adf5611.html>`_ or `EVAL-ADF5612 <https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/eval-adf5612.html>`_
+* `SDP-K1 <https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/sdp-k1.html>`_
+
+**Connections**:
+
+The Arduino of the SDP-K1 needs to be connected to Arduino header of the 
+Evaluation board.
+Additionally a 6V power supply needs to be connected to either J4 
+(the SMA interface).
+
+**Build Command**
+
+.. code-block:: bash
+
+	# add the arm gcc to the PATH variable
+	export PATH=</path/to/arm/gcc/bin>:$PATH
+	# to delete current build
+	make reset
+	# to build the project
+	make 
```

Comment:
also, with the new project example changes, provide the commands `make EXAMPLE=basic` and `make EXAMPLE=iio_example`